### PR TITLE
[cryptopp] update to v8_5_0

### PR DESCRIPTION
**Pull request by hros**

### CODE REVIEW

File: C/cryptopp/build_tarballs.jl [hunk 1]
```diff
@@ -7,7 +7,7 @@ version = v"8.2"
 
 # Collection of sources required to build CFITSIO
 sources = [
-    GitSource("https://github.com/weidai11/cryptopp.git", "9dcc26c58213abb8351fbb1b2a7a1d2c667366e4"),
+    GitSource("https://github.com/weidai11/cryptopp.git", "f2102243e6fdd48c0b2a393a0993cca228f20573"),
```
Comment (giordano) [inline C/cryptopp/build_tarballs.jl:10]: You didn't change the version number

**Comment by giordano** [comment]
I made a couple of changes:

* the new JLL package will require Julia v1.6
* this recipe now only builds the shared library (the only product listed anyways), and not the `cryptotest` program, too.

Can you please confirm whether you're happy with these changes?

**Comment by giordano** [comment]
Ok, I'm assuming the answer to the question above is "yes" and I'm going to merge this :wink:

**Comment by hros** [comment]
Sorry for missing your question
And, naturally, the answer is:yes.

And thanks for merging the package

**Pull request merged**